app/Http/Controllers/ProjectController.php

```diff
@@ -114,12 +114,22 @@ public function show(Project $project): View
 
     public function edit(Project $project): View
     {
+        $orientationLink = '#';
+        if (Auth::user()->context === 'regulated-organization') {
+            $orientationLink = 'https://share.hsforms.com/1gGf9TjhaQ0uaqcnyJfSDlwdfpez';
+        } elseif (Auth::user()->context === 'organization') {
+            $orientationLink = 'https://share.hsforms.com/1sB6UV4gvQlC_0QxQ3q3z1Adfpez';
+        } elseif (Auth::user()->context === 'individual') {
+            $orientationLink = 'https://share.hsforms.com/161eyaBsQS-iv1z0TZLwdQwdfpez';
```

Comment:
An alternative way to write this would be to use [match](https://www.php.net/manual/en/control-structures.match.php). If we update this again we should also consider moving this to somewhere that can be reused to so that we can update the links in a single location if needed again.

Reply:
Actually now that there's a merge conflict can you update this to use the match statement. Also consider moving this to a helper function or something else and making the call to it in the other locations. You could probably also write a test for the logic then. Another option instead of a helper is an Enum. There may be other ways to generalize it too.